CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Cromwell Change Log
 
+## 29
+
+### Breaking Changes
+
+* Request timeouts for HTTP requests on the REST API now return a 503 status code instead of 500. It also no longer contains a JSON response.
```

Comment:
Maybe "The response for a request timeout is no longer in JSON format"?  The antecedent of the "It" seems a little ambiguous.